package.json

```diff
@@ -21,7 +21,7 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.13.0"
+    "react": "^0.13.0 || ^0.14.0"
```

Comment:
Ok I'v set it to 0.14 now. Turned out it had to be done anyway, because findDOMNode is depricated. I have added the react-dom dependency to solve it.